Components/badge/badge.css

```diff
@@ -0,0 +1,94 @@
+.VariantContainer
+{
+      margin-top: 25px;
+      border: 3px solid var(--primary-color);
+      width: 900px;
+      height: 200px;
+      display: flex;
+      flex-direction: row;
+      flex-wrap: wrap;
+      gap: 25px;
+      justify-content: center;
+      align-items: center;  
+      text-align: center; 
+}   
+
+#badge-container p span{
```

Comment:
id's specificity is too high, use it only when you donot have any other option, always keep the specificity as low as possible